crates/ln-dlc-node/src/lib.rs

```diff
@@ -142,6 +143,7 @@ impl From<Invoice> for PaymentInfo {
                 InvoiceDescription::Direct(direct) => direct.to_string(),
                 InvoiceDescription::Hash(hash) => hash.0.to_hex(),
             },
+            invoice: Some(format!("{value}")),
```

Comment:
🔧 A bit simpler?
```suggestion
            invoice: Some(value.to_string()),
```

Reply:
Right, I forgot about that trait impl :)